CHANGELOG.md

```diff
@@ -15,6 +15,15 @@
   * `locales` has been removed from site config. As of 6.23 Locales are pulled from the global `TERRA_AGGREGATED_LOCALES` variable provided by terra-toolkit v5.21.0.
   * Changed to a scoped package. `terra-dev-site` -> `@cerner/terra-dev-site`
   * Remove defaulting to devtool: source-map when in production with no devtool enabled. This was causing slow builds for large projects.
+  * Terra-dev-site fully converted to a webpack plugin.
```

Comment:
Are these all truly breaking?  It might be helpful for consumers to know what the specific breaking changes are.

Reply:
It's debatable, but i'll break some of these out.